src/users/components/UserList.vue

```diff
@@ -117,6 +124,9 @@ export default {
     inactiveSublabel () {
       return this.inactiveUsers.length + ' ' + this.$tc('JOINGROUP.NUM_MEMBERS', this.inactiveUsers.length)
     },
+    activeUsersCount () {
```

Comment:
Here it's called "activeUsersCount", but in the [video from @tiltec](https://github.com/yunity/karrot-frontend/pull/2149#issuecomment-673380955) it looks like it's counting _all_ the members (5 active + 1 inactive).

I'm not actually sure which is a more useful number? Although given we already have the inactive user count in the little Inactive section, maybe active members is most useful? and given you named it "activeUsersCount" maybe that is what was intended too.

In which case you can use `this.activeUsers.length` (instead of `this.users.length`).

A nitpick too, I would expect something named "activeUsersCount" to return me a _number_ rather than the text, so maybe `activeUsersLabel` is more descriptive?

Alternatively there is the template syntax, which could look something like this:

```html
<QItem v-t="{ path: 'GROUP.MEMBERLIST', args: { total: activeUsers.length } }" />
```